main.sh

```diff
@@ -257,12 +258,11 @@ for test in "$@"; do
 	    echo "SELF_TEST is temporarily disabled - DAOS-3838"
             ;;
         MDTEST)
```

Comment:
Change the name to MDTEST_EASY if there is plan to add HARD in future?

Reply:
Previous options would not have worked so just to have a base set of working options I included mdtest-easy options. But no such plan to add hard params at least from my side for now.